invenio_indexer/api.py

```diff
@@ -164,12 +164,13 @@ def delete_by_id(self, record_uuid):
         """Delete record from index by record identifier."""
         self.delete(Record.get_record(record_uuid))
 
-    def bulk_index(self, record_id_iterator):
+    def bulk_index(self, record_id_iterator, index=None, doc_type=None):
```

Comment:
As discussed IRL, the best approach is to ensure ``$schema`` attribute is present in the record (i.e. please revert this change)